exercism/hamming/hamming.js

```diff
@@ -0,0 +1,16 @@
+var Hamming = function() {}
+
+Hamming.prototype.compute = function(str1, str2) {
+
+  if (str1.length !== str2.length) {
+    throw new Error('DNA strands must be of equal length.')
+  }
+  var count = 0
+  for (var i = 0; i < str1.length; i++) {
+    if (str1[i] !== str2[i]) {
```

Comment:
If this is a DNA problem I think 'A' only are supposed to match with 'T' and 'G' with 'C'.